e2e/constants.ts

```diff
@@ -11,8 +11,8 @@ export const CONTACT_PAGE_ENDPOINT = '/contact';
 export const INVALID_EMAIL = 'test_email@invalid';
 export const USER_NAME = 'test';
 export const BRAND_NAME = process.env.PLAYWRIGHT_BRAND_NAME;
-export const SIGN_IN_PAGE_ENDPOINT = '/auth/signin';
-export const SIGN_UP_PAGE_ENDPOINT = '/auth/create';
+export const SIGN_IN_PAGE_ENDPOINT = '/account/signin';
```

Comment:
Consolidated all auth urls to be in `/account`, to follow the Shopify convention.